src/main/java/io/jenkins/plugins/audit/listeners/BuildStartListener.java

```diff
@@ -37,7 +39,7 @@ public void onStarted(Run run, TaskListener listener) {
         buildStart.setBuildNumber(run.getNumber());
         buildStart.setCause(causes);
         buildStart.setProjectName(run.getParent().getFullName());
-        buildStart.setTimestamp(new Date(run.getStartTimeInMillis()).toString());
+        buildStart.setTimestamp(formatDateISO(new Date(run.getStartTimeInMillis())));
```

Comment:
As we are using Java 8, it would be more appropriate to use one of these two datetime formatters:

* https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME
* https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_OFFSET_DATE_TIME

Reply:
@jvz, Thanks for your comments and help in me knowing this. I have updated them after learning about a lot of confusions with the date time formats and daylight savings in the other part of the globe.

Reply:
Dates and timestamps are an extraordinarily complicated topic. No worries!